src/manage_nsfs/manage_nsfs_help_utils.js

```diff
@@ -296,37 +412,51 @@ List of actions supported:
-But updates of the config directory will be blocked during the upgrade of the config directory.
-'upgrade start' should be executed on one node, the config directory changes will be available for all the nodes of the cluster.
+Help:
+
+    'upgrade start' is a noobaa-cli command that will start config directory upgrade run.
+    Run 'upgrade start' after upgrading NooBaa RPMs on all the cluster nodes, after starting an upgrade of the config directory,
+    S3 I/O, S3 Buckets getters and NooBaa CLI Account/Buckets/Whitelist getters operations will still be working
+    But updates of the config directory will be blocked during the upgrade of the config directory.
+    'upgrade start' should be executed on one node, the config directory changes will be available for all the nodes of the cluster.
 
 Usage:
         
     noobaa-cli upgrade start [flags]
 
 Flags:
 
---expected_version                             <string>                             The expected target version of the upgrade
---expected_hosts                               <string>                             The expected hosts running NooBaa NC, a string of hosts separated by , 
---skip_verification                            <boolean>        (optional)          skip verification of the hosts package version
-                                                                                    WARNING: can cause corrupted config dir files created by hosts running old code
---custom_upgrade_scripts_dir                   <string>         (optional)          custom upgrade scripts dir, use for running custom config dir upgrade scripts
+    --expected_version                             <string>                             The expected target version of the upgrade
+    --expected_hosts                               <string>                             The expected hosts running NooBaa NC, a string of hosts separated by , 
+    --skip_verification                            <boolean>        (optional)          skip verification of the hosts package version
+                                                                                        WARNING: can cause corrupted config dir files created by hosts running old code
+    --custom_upgrade_scripts_dir                   <string>         (optional)          custom upgrade scripts dir, use for running custom config dir upgrade scripts
 
 `;
 
 const UPGRADE_STATUS_OPTIONS = `
-'upgrade status' is a noobaa-cli command that will return the status of an ongoing upgrade run,
-the available status information is upgrade start timestmp, from_version, to_version, config_dir_from_version,
-config_dir_to_version, running_host etc.
+Help:
+
+    'upgrade status' is a noobaa-cli command that will return the status of an ongoing upgrade run,
+    the available status information is upgrade start timestmp, from_version, to_version, config_dir_from_version,
+    config_dir_to_version, running_host etc.
+
+Usage:
+
+    noobaa-cli upgrade status
```

Comment:
@romayalon we don't have any flag? (not even those in `CLI_MUTUAL_FLAGS`? `config_root`, `config_root_backend`, `debug`?)
Same question for `upgrade start` and `upgrade history`

Reply:
moving it to another issue : https://github.com/noobaa/noobaa-core/issues/8595